bc_obps/registration/api.py

```diff
@@ -0,0 +1,134 @@
+from django.contrib import admin
+from django.urls import path
+from datetime import date
+from typing import List
+from ninja import Router
+from django.shortcuts import get_object_or_404
+from .models import Operation, Operator, NaicsCode
+from ninja.orm import create_schema
+from ninja import Field, Schema
+from decimal import *
+
+
+router = Router()
+
+
+# Naics code schemas and endpoints
+class NaicsCodeSchema(Schema):
+    id: int
+    naics_code: str
+    ciip_sector: str
+    naics_description: str
+
+
+@router.get("/naics_codes", response=List[NaicsCodeSchema])
+def list_naics_codes(request):
+    qs = NaicsCode.objects.all()
+    return qs
+
+
+# Operation schemas and endpoints
+class OperationIn(Schema):
+    operator: int = Field(..., alias="operator_id")
+    name: str
+    operation_type: str
+    naics_code: int = Field(..., alias="naics_code_id")
+    eligible_commercial_product_name: str
+    permit_id: str
+    npr_id: str
+    ghfrp_id: str
+    bcghrp_id: str
+    petrinex_id: str
+    latitude: Decimal
+    longitude: Decimal
+    legal_land_description: str
+    nearest_municipality: str
+    operator_percent_of_ownership: Decimal
+    registered_for_obps: bool
+    estimated_emissions: Decimal
+    registered_for_obps: str = Field(default=False)
+    # contacts:
+    # documents:
```

Comment:
These will be handled in https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/138